src/plugins/data/public/ui/saved_query_flyouts/open_saved_query_flyout.tsx

```diff
@@ -61,7 +61,9 @@ export function OpenSavedQueryFlyout({
   const [searchQuery, setSearchQuery] = useState(EuiSearchBar.Query.MATCH_ALL);
 
   const fetchAllSavedQueriesForSelectedTab = useCallback(async () => {
-    const allQueries = await savedQueryService.getAllSavedQueries();
+    // const allQueries = await savedQueryService.getAllSavedQueries();
+    // TODO: Mitigation until getAllSavedQueries gets fixed
+    const { queries: allQueries } = await savedQueryService.findSavedQueries('', 10000);
```

Comment:
Is this behind a feature flag or would it be used in either case? Is there a risk with the 10000 limit (I'm assuming)?

Reply:
It is behind flag and we will get it fixed as a fast follow  and we don't see a risk with count as 10k
That's the default upper limit